client/tests/integration/add_domain.rs

```diff
@@ -18,22 +18,18 @@ fn client_add_domain_with_name_length_more_than_limit_should_not_commit_transact
 
     // Given
 
-    let normal_domain_name = DomainId::new("sora")?;
-    let create_domain = RegisterBox::new(IdentifiableBox::from(Domain::new(
-        normal_domain_name.clone(),
-    )));
+    let normal_domain_id: DomainId = "sora".parse()?;
```

Comment:
While I think this type can be inferred, having more explicit type annotations doesn't hurt.